server/views/pages/adjustments/remand/partials/remandOffence.njk

```diff
@@ -24,3 +24,40 @@
     Offence date not entered
   {% endif %}
 {% endmacro %}
+
+{% macro offencesCard(sentence, offences) %}
+  <h2 class="govuk-heading-m">Court case {{sentence.caseSequence}}</h2>
+  <div class="govuk-summary-card govuk-!-margin-bottom-7">
+    <div class="govuk-summary-card__content equal-padding">
+      <p class="govuk-body">
+        <strong>
+          Court name
+        </strong>
+        {{ sentence.courtDescription }}
+        <br/>
+        <strong>
+          Case reference
+        </strong>
+        {{ sentence.caseReference or 'Not entered' }}
+      </p>
+      <p class="govuk-body govuk-!-margin-top-1 govuk-!-margin-bottom-2">
+        <strong>
+          Offences
+        </strong>
+        <br/>
+        <span class="govuk-hint">Select all that apply</span>
+      </p>
+      {% for offence in offences %}
+        <div class="govuk-checkboxes__item">
```

Comment:
have you considered using https://design-system.service.gov.uk/components/checkboxes/ component for this? I think using the nunjucks function for the component gives then benefit if they change the component in the future you'd get the change for free upgrading, using straight html doesn't give that benefit.

Reply:
It's not really feasible in the context of a macro because of the amount of dynamic content in the checkbox label.
Whilst I understand your concern that using the Nunjucks function offers the benefit of automatically updating the component, GDS updates are typically designed to be backwards compatible, and they take into account the needs of various governmental departments to ensure that updates don't disrupt service. Plus, their focus on accessibility means they're always aiming to enhance, not hinder, the usability of components across different devices and user needs.
This priority on stability, accessibility, and compatibility across devices makes It is highly unlikely that they'd roll out an update that completely wrecks existing HTML implementations, especially since many government apps depend on the robustness of these implementations.